packages/editor/package.json

```diff
@@ -11,7 +11,7 @@
     "clipboard": "^2.0.1",
     "color": "^3.0.0",
     "common": "0.0.1",
-    "custom-functions-metadata": "^0.0.19",
+    "custom-functions-metadata": "0.0.22",
```

Comment:
I would prefer this have a "^" in front of it, that way `yarn interactive-update` will pick up new versions in the future.

Reply:
I followed notes to use: 
```
yarn workspace editor upgrade custom-functions-metadata@0.0.22
```
I don't use yarn that much. Is there a better command to use? Do i need to specify `custom-functions-metadata@^0.0.22` or is there some other way to get the caret along with the `latest` tag version?

Reply:
Because the version number is 0.0.x, the semver using caret doesn't make a difference. We should probably go ahead and update custom-functions-metadata to 1.0.0, and then the semver updates would happen.

Reply:
I *think* if you pass the caret on the commandline, then yeah, it should add the ^ in the package.json and the yarn.lock as well.
But if you are in the state where you are now, you can just manually update those two files in those two lines and call it good.

Reply:
Oh, I see.  Maybe.  I thought it's worked for me for other stuff before but not always, maybe that explains why.
In the past, if I didn't end up with a caret when needed, I would just add it manually to package.json and yarn.lock.

Reply:
It doesn't make any difference for 0.0.x. There's no point to do it, as 0.0.23 doesn't match ^0.0.22.

Reply:
I'm 99% sure that having the "^" does make a difference for "yarn interactive-upgrade" though.

Reply:
And it keeps it semantically consistent with the intent (of "I want the latest cf metadata parser", as opposed to I had a specific reason to lock in to version x.y.z of something like we do with `"query-string": "5.1.1",`)

Reply:
I understand the intent. However, it would really be better to start at version 0.1.0 so that caret means something. It will look like it means something however for 0.0.x it really means nothing different with the caret from without the caret. I'm looking into bumping custom-functions-runtime to version 0.1.0 or 1.0.0...    but I might wait until we merge some other changes. Is this urgent?

Reply:
Yeah, ok, that's fine.  I'll go ahead and merge in as-is, then, and we can fix it up the next time you bump up the version.